lib/customer.rb

```diff
@@ -0,0 +1,50 @@
+require_relative 'take_away'
+require_relative 'send-sms'
+
+class Customer
+  attr_reader :m_takeaway
+  attr_reader :basket
+  attr_reader :total
+  attr_reader :sms
+  def initialize (m_takeaway = Takeaway.new,sms = Sms.new)
+    @basket = []
+    @total = 0
+    @m_takeaway = m_takeaway
+    @sms = sms
+    
+  end
+
+  def readmenu
+    m_takeaway.list_menu
+  end
+
+  def add_item(item)
```

Comment:
I really like how you've constructed the add and remove methods using simple syntax, instead of looping through the menu hash